plugin.video.fanfilm/resources/lib/sources/pl/filiser.py

```diff
@@ -162,13 +175,13 @@ def extract_sources(self, transl_type, links, typ, html):
             elif quality.endswith('1080p'):
                 q = '1080p'
 
-            sources.append({'source': host, 'quality': q, 'language': lang, 'url': (typ,code,code2,data_refs[i]), 'info': info, 'direct': False, 'debridonly': False})
+            sources.append({'source': host, 'quality': q, 'language': lang, 'url': (typ,code,code2,data_refs[i],title,title2), 'info': info, 'direct': False, 'debridonly': False})
 
         return sources
 
     def resolve(self, url):
         try:
-            url_to_exec = urlparse.urljoin(self.base_link, self.url_transl) % (url[0],url[1],url[2],url[3])
+            url_to_exec = urlparse.urljoin(self.base_link, self.url_transl) % (url[0],url[1],url[2],url[3],url[4],url[5])
```

Comment:
Będzie działać, ale mógłbyś to zmienić na:
```python
    url_to_exec = urlparse.urljoin(self.base_link, self.url_transl) % url
```

Bardziej czytelne mi się wydaje. Stare formatowanie w miejsce każdego `%` wstawia kolejny argument (pomijam argumenty nazwane). Czyli w Twoim kodzie tworzysz tuple z... ręcznie wpisanych elementów tuple (lub listy) `url`, co jest zdecydowanie nadmiarowe.

Reply:
Sprawdziłem u siebie na 16.4 i seriali mi nie odtwarza z % url, źle składa adres, po rozbiciu filmy i seriale ładuje OK

początkuję w pythonie i nie wiem na 100% czemu tak się dzieje, ale zdaje sie że przy serialach zmienia się liczba parametrów przekazywanych

Reply:
Niewiarygodne! Sprawdzę później. Teraz łykam tak, jak napisałeś. Dzięki.

Reply:
Mea maxima culpa. Powinienem sprawdzić dokładniej.

Zgaduję, że tam jest słownik (dict) i dlatego nie działa. Są jeszcze dwa inne szybkie sposoby na rozwiązanie tego. Ale niech na razie zostanie Twój pierwotny.
Daj jeszcze PR. To klepnę za 3 min, jak nie to w czwartek (jutro jest z dala od świata), albo poproś @xulek, to klepnie w wolnej chwili. 
Przepraszam za zamieszenie.